### PR TITLE
Makefile: Add ginkgo test-fast

### PR DESCRIPTION
**Pull request by eloycoto**
This commits depends on PR #28, that adds ginkgo v2 support.

This is a small change on Makefile to run ginkgo directly, so with this
change, in future changes when Kind/e2e jobs are in place, we can use
gingko labels[0] to execute the right test[1].

[0] https://onsi.github.io/ginkgo/#spec-labels
[1] go test runner does not support ginkgo labels.

Signed-off-by: Eloy Coto <eloy.coto@acalustra.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -306,3 +311,10 @@ catalog-build: opm ## Build a catalog image.
 .PHONY: catalog-push
 catalog-push: ## Push a catalog image.
 	$(MAKE) docker-push IMG=$(CATALOG_IMG)
+
+GINKGO = $(shell pwd)/bin/ginkgo
+.PHONY: ginkgo
+ginkgo: ## Download ginkgo locally if necessary.
+ifeq (, $(wildcard $(GINKGO)))
+	$(call go-install-tool,$(GINKGO),github.com/onsi/ginkgo/v2/ginkgo@v2.1.3)
```
Comment (bardielle) [inline Makefile:319]: Please update also go.mod file to include ginkgo version 2

Comment (eloycoto) [reply Makefile:319]: As I mentioned, the go.mod is part of the PR #28

Comment (bardielle) [reply Makefile:319]: I think it's better to include it as part of that PR, I have a  PR with UT that includes changes in the Makefile, vendor, and go.mod file

Comment (ygalblum) [reply Makefile:344]: V2 is included from #28 and removed here. @bardielle can we consider this conversion resolved?

**Review by ygalblum — APPROVED** [review]

**Pull request merged**